tModPorter/tModPorter.Tests/TestData/GlobalItemTest.Expected.cs

```diff
@@ -1,7 +1,19 @@
 using Terraria;
 using Terraria.ModLoader;
+using Terraria.ModLoader.IO;
+using Microsoft.Xna.Framework;
```

Comment:
Additional framework import? Probably just a leftover.

Reply:
Rectangle for UseStyle